tests/rptest/tests/e2e_shadow_indexing_test.py

```diff
@@ -302,4 +302,144 @@ def create_or_delete_until_producer_fin():
 
         producer.wait()
         rand_consumer.shutdown()
-        rand_consumer.wait()
\ No newline at end of file
+        rand_consumer.wait()
+
+
+class ShadowIndexingConsumerGroupsTest(RedpandaTest):
+    # Run a workload against SI with group consuming.
+    # In the past, a customer has run into issues with group consuming on systems
+    # with SI enabled
+    # This class ports Test7 from https://github.com/redpanda-data/redpanda/issues/3572
+    # into ducktape.
+    segment_size = 1048576
+    num_groups = 2
+    groups = [f'group-{i}' for i in range(num_groups)]
+    topics = [TopicSpec(partition_count=5) for i in range(num_groups)]
+
+    # 1k messages of size 2**15
+    # is ~32MB of data. Using ~32MB of data
+    # because RpkConsumer fails at
+    # GBs of data.
+    msg_size = 2**15
+    msg_count = 1000
+
+    def __init__(self, test_context):
+        self._si_settings = SISettings(
+            log_segment_size=self.segment_size,
+            cloud_storage_cache_size=20 * 2**30,
+            cloud_storage_reconciliation_interval_ms=500,
+            cloud_storage_max_connections=5,
+            cloud_storage_enable_remote_read=False,
```

Comment:
I think this test uses SI on all topics, right?  So it would make sense to enable remote_[read|write] here